app/views/campaign_requests/_request_details.html.erb

```diff
@@ -3,15 +3,8 @@
 </div>
 
 <p>
-  Use this form for : Campaign Platform site or microsite
-<ul>
-  <li>A campaign platform site is a one page website on campaign.gov.uk</li>
-  <li>Microsite websites are not built on GOV.UK and may or may not have
-    a gov.uk url.
-    Microsite websites <%= link_to 'fall under Cabinet Office digital service expenditure', 'https://www.google.com/url?q=https://www.gov.uk/service-manual/agile-delivery/spend-controls-check-if-you-need-approval-to-spend-money-on-a-service%23when-your-service-is-considered-a-digital-service&sa=D&ust=1518444538536000&usg=AFQjCNFvnZ81hn7cqJnGHgcgQ0V0bdwCBQ'%>
-    therefore GDS spend approval must be sought following approval from GCS to develop a campaign microsite.
-  </li>
-</ul>
+  Use this form for : Campaign Platform site or a multi-page campaign <br>
```

Comment:
I know it was already there, but would you be able to remove the space between `for` and the `:`?

I think it might be worth going back to product by commenting on the Trello ticket or messaging Lisa to find out if they have any preference for the content... I know I am not a content designer 😅 but I think something along the lines of:

```
<p>
  Use this form for a campaign platform site or a multi-page campaign.
</p>

<p>
  For either type of site, please answer the questions below.
</p>
```

would work better. I also think we should remove the `br` tags. It looks a bit tidier. Semantically they could be in the same paragraph but I think it also works having them separate. Having them separate stops the text wrapping around on a smaller screen which reads a bit better.

If it will take too long for the content question to be answered then:

```
<p>
  Use this form for: Campaign Platform site or a multi-page campaign.
</p>
<p>
  For either type of site, please answer the questions below.
</p>
```

and we can update it at a later date.

Reply:
Thanks for the review Rosa, now using 2 <p> as you recommended. I used br initially as I thought 2 <p> seemed heavy handed, but I hadn't thought of the wrapping issue 😅 .